src/utils/json.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <Poco/JSON/Object.h>
+#include <Poco/JSON/Parser.h>
+
+#include "utils/json.hpp"
+
+namespace UtilsJson {
+
+Poco::Dynamic::Var ParseJson(const std::string& jsonStr)
+{
+    auto parser = Poco::JSON::Parser();
+
+    return parser.parse(jsonStr);
```

Comment:
I'd keep throw logic `cause it simplifies error handling.
We can add an  overload that will not throw if needed.

Reply:
@mlohvynenko  i'm not sure that it could simplify logic because when I call this function I don't what type of exception could be thrown by this function and so on. In my opinion, it's better to use thrown exceptions only in local modules but not external, ones because it's hard to understand what exception on the caller side I need to handle

Reply:
Refactored to return error